ADALiOS/ADALiOS.xcodeproj/project.pbxproj

```diff
@@ -461,18 +465,14 @@
 			children = (
 				6071B5E21C14C0B0006F6CC2 /* ADTestURLConnection.h */,
 				6071B5E31C14C0B0006F6CC2 /* ADTestURLConnection.m */,
-				8B6113FC1868C59E007759C2 /* ADTestAuthenticationContext.h */,
-				8B6113FD1868C59E007759C2 /* ADTestAuthenticationContext.m */,
-				8BB8346A1807BE3F007F9F0D /* ADAuthenticationContextTests.m */,
+				8BB8346A1807BE3F007F9F0D /* ADAcquireTokenTests.m */,
+				9430C3551C55862A00D6506D /* ADAuthorityValidationTests.m */,
+				9430C34D1C54320400D6506D /* ADAuthenticationContextTests.m */,
 				8BB8346C1807C5F5007F9F0D /* ADAuthenticationParametersTests.m */,
 				D6C38DC91BA0E74F00D62ABF /* ADBrokerKeyHelperTests.m */,
 				8B0965C417F25770002BDFB8 /* Supporting Files */,
-				8B598960181710FD00744AEE /* ADDefaultTokenCacheStoreTests.m */,
+				8B598960181710FD00744AEE /* ADKeychainTokenCacheStoreTests.m */,
```

Comment:
Given that "ADKeychainCacheStore" being changed to"ADKeychainCache", shouldn't this be "ADKeychainCacheTests"? :P